src/Conversation.swift

```
@@ -185,7 +428,7 @@ public extension Conversation {
 		if !handlingVoice { try startHandlingVoice() }
 
 		Task.detached {
-			self.audioEngine.inputNode.installTap(onBus: 0, bufferSize: 4096, format: self.audioEngine.inputNode.outputFormat(forBus: 0)) { [weak self] buffer, _ in
+			self.audioEngine.inputNode.installTap(onBus: 0, bufferSize: 4096, format: self.audioEngine.inputNode.inputFormat(forBus: 0)) { [weak self] buffer, _ in
```

Call removeTap(onBus:) before installing a new tap; otherwise the original tap leaks and silently drops buffers.